docs/src/appendix/cheatcodes/start_spoof.md

```diff
@@ -0,0 +1,71 @@
+# `start_spoof`
+
+> `fn start_spoof(contract_address: ContractAddress, tx_info_mock: TxInfoMock)`
+Changes `TxInfo` returned by `get_tx_info()` for the targeted contract until the spoof is stopped
+with [stop_spoof](./stop_spoof.md).
+
+- `contract_address` address of the contract for which `get_tx_info()` result will be mocked.
+- `TxInfoMock` - a struct with same structure as `TxInfo` (returned by `get_tx_info()`), 
+
+To mock the field of `TxInfo`, set the corresponding field of `TxInfoMock` to `Some(mocked_value)`. Setting the field to `None` will use a default value - the field will not be mocked. Using `None` will also cancel current mock for that field. See below for practical example.
```

Comment:
We should add an example about using `None` as a cancel of the spoof

Reply:
added in https://github.com/foundry-rs/starknet-foundry/pull/482/commits/17bb23850739c6f871a542eae54947828c146bb0